### PR TITLE
layer2 store operator and server

### PR DESCRIPTION
**Pull request by blockchain-develop**

### CODE REVIEW

File: operator/core/operator.go [hunk 1]
```diff
@@ -647,11 +553,12 @@ func (this *Layer2Operator) commitLayer2State2Ontology(msg *Layer2CommitMsg) err
...
 	var gasLimit uint64
 	if err != nil {
-		gasLimit = 6000000
+		log.Warnf("Pre exec contract err: %s", err.Error())
+		gasLimit = this.config.OntologyConfig.GasLimit
 	} else {
 		gasLimit = result.Gas
 	}
-	tx, err := this.ontologySdk.NeoVM.NewNeoVMInvokeTransaction(500, gasLimit, contractAddress, []interface{}{"updateState", []interface{}{
+	tx, err := this.ontologySdk.NeoVM.NewNeoVMInvokeTransaction(this.config.OntologyConfig.GasPrice, gasLimit, contractAddress, []interface{}{"updateState", []interface{}{
```
Comment (laizy) [inline operator/core/operator.go:561]: support batch update, it will get little benefit if constructing a transaction for every layer2 block.

Comment (blockchain-develop) [reply operator/core/operator.go:561]: if there is no batch. The highest performance is 1000 ~ 2000.

Comment (blockchain-develop) [reply operator/core/operator.go:561]: If there are much transactions in a block, batch is good.
If there is only one transaction in a block, batch is not bad.

Comment (laizy) [reply operator/core/operator.go:561]: there is nothing to do with performance, batching is needed to reduce operator's gas fee.

Comment (blockchain-develop) [reply operator/core/operator.go:561]: so......  batch is needed!

Comment (blockchain-develop) [reply operator/core/operator.go]: updated

**Pull request merged**